crypto/rand/rand_lib.c

```diff
@@ -187,7 +196,7 @@ const RAND_METHOD *RAND_get_rand_method(void)
     return tmp_meth;
 }
 
-# if !defined(OPENSSL_NO_ENGINE)
+# if !defined(OPENSSL_NO_ENGINE) && !defined(OPENSSL_NO_DEPRECATED_3_0)
```

Comment:
Please break it in two lines:

``` C
# ifndef OPENSSL_NO_ENGINE
#  ifndef OPENSSL_NO_DEPRECATED_3_0
```

OpenSSL::ParseC isn't smart enough to discern ordinal file feature sets from too complex preprocessor expressions...

Reply:
sure, done.